docs/managing-e2e-tests.md

```diff
@@ -23,106 +24,140 @@ It contains a list of jobs such as:
+In the YAML structure of jobs such as `ci-kubernetes-e2e-kubeadm-kind-master` the following list
+of annotations can be seen:
 
-Tests can exist in multiple dashboards at once. For example a test defined like:
 ```
-- name: kubeadm-kind-1-12
-  test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-12
+annotations:
+  testgrid-dashboards: sig-cluster-lifecycle-kubeadm,sig-release-master-informing
+  testgrid-tab-name: kubeadm-kind-master
+  testgrid-alert-email: kubernetes-sig-cluster-lifecycle@googlegroups.com
+  description: "OWNER: sig-cluster-lifecycle (kind); Uses kubeadm/kind to create a cluster and run the conformance suite"
+  testgrid-num-columns-recent: "20"
+  testgrid-num-failures-to-alert: "4"
+  testgrid-alert-stale-results-hours: "8"
 ```
-can exists in both the `sig-cluster-lifecycle-all` and `sig-release-master-blocking` dashboards. Essentially both tests would show details about the same test job called `ci-kubernetes-e2e-kubeadm-kind-1-12`.
+
+These annotations configure the testgrid entries for this job from the job configuration itself.
+They contain information such as dashboards where this job will appear, tab name, where to send email alerts,
+description and other.
+
+For more information about configuring testgrid see [this page](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md).
 
 ### Updates to kubeadm tests
 
 Before each new Kubernetes release and during the second month of the [release-cycle](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md), a set of manual actions have to be performed, so that the kubeadm e2e tests are up to date with the new release.
 
-The operation can be broken down in 3 steps:
-1) Updating the kubeadm test jobs and parameters.
-2) Updating testgrid to reflect the changes.
-3) Finalizing the changes.
+The operation can be broken down into:
+- Sending a PR for updating kinder workflows.
+- Sending a PR for updating test jobs in kubernetes/test-infra.
+
+Ideally both PRs should be merged around the same time, so that there are no failed runs.
 
-There is work in progress in test-infra on tools that can hopefully automate this process.
+The idea is to remove old tests and add new ones. For example. if Kubernetes 1.15 is about to be released soon,
+this means that the `15 - 3 = 12` MINOR version would go outside of the support skew of the project. `1.12` jobs
+have to be removed and `1.15` jobs have to be added. The project would support version `15 - 2 = 13` as the minimum.
 
-#### Updating test jobs
+#### Updating kinder workflows
+
+Kinder is deployment tool that currently resides in the kubernetes/kubeadm repository.
+It contains a folder with workflows that need update each cycle:
+https://github.com/kubernetes/kubeadm/blob/master/kinder/ci/workflows/
+
+The summary of the actions is the following:
```

Comment:
before the list, might be worth to mention the assumptions: 
if Kubernetes 1.15 is about to be released soon (and 1.12 version would go outside of the support),

Reply:
the assumptions are in the previous section `Updates to kubeadm tests` and they apply to both updating test-infra and kinder workflows.